README.md

```diff
@@ -54,6 +54,10 @@ Some programming experience is also recommended.
     Modern NLP techniques from recurrent neural networks and word embeddings
     to transformers and self-attention. Covers applied topics like questions answering and 
     text generation.
+    
+* [Deep Learning - NYU - 2000](https://www.youtube.com/playlist?list=PLLHTzKZzVU9eaEyErdV26ikyolxOsz6mq) | [Course website](https://atcold.github.io/pytorch-Deep-Learning/)
+
+    This course concerns the latest techniques in deep learning and representation learning, focusing on supervised and unsupervised deep learning, embedding methods, metric learning, convolutional and recurrent nets, with applications to computer vision, natural language understanding, and speech recognition. The prerequisites include: DS-GA 1001 Intro to Data Science
```

Comment:
The course certainly looks good. However, I would rather not mention the prerequisite course, since almost nobody reading this description will be familiar with it.

Reply:
removed prerequisite